storage_files/books.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
- [ OPTIONAL ] Great job for displaying the results but as I could see some extra slashes but to have a clear vesion I would highly recommend to just remove them from the files 🙏🏻✅✨💯

![image](https://user-images.githubusercontent.com/58553711/225166221-f2f02da5-4250-4a66-9890-a8665edc4ffe.png)

Reply:
Thank you for the suggestion, I work on it as soon as possible  👍🏾